tests/h/form_test.py

```diff
@@ -164,15 +164,16 @@ def test_does_not_show_flash_message_if_xhr(self, pyramid_request):
 @pytest.mark.usefixtures('to_xhr_response')
 class TestHandleFormSubmission(object):
 
-    def test_it_calls_validate(self, pyramid_request):
+    def test_it_calls_validate(self, pyramid_request, matchers):
         form_ = mock.Mock(spec_set=['validate'])
 
         form.handle_form_submission(pyramid_request,
                                     form_,
                                     mock_callable(),
                                     mock.sentinel.on_failure)
 
-        form_.validate.assert_called_once_with(pyramid_request.POST.items())
+        post_items = matchers.iterable_with(list(pyramid_request.POST.items()))
+        form_.validate.assert_called_once_with(post_items)
```

Comment:
Note that `POST.items()` returns an iterable but `matchers.iterable_with` requires a list rather than an iterable as an argument.

The `POST` dict is in fact empty in this test, which makes this not a great test, but I left it as-is and just fixed the compat issue.